var/spack/packages/openblas/package.py

```diff
@@ -0,0 +1,24 @@
+from spack import *
+
+class Openblas(Package):
+    """OpenBLAS: An optimized BLAS library"""
+    homepage = "http://www.openblas.net"
+    url      = "http://github.com/xianyi/OpenBLAS/archive/v0.2.15.tar.gz"
+
+    version('0.2.15', 'b1190f3d3471685f17cfd1ec1d252ac9')
+
+    # virtual dependency
+    provides('blas')
+    provides('lapack')
```

Comment:
Can you do a `spack find -d <hypre-spec>` on the resulting install?  Does it end up building a separate `blas` or a separate `lapack`, or does `openblas` satisfy both?